pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -4046,6 +4049,45 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Entry("no hyperv use", doNotUseExplicitHyperV, doNotUseHyperVPassthrough, true),
 		)
 	})
+
+	It("when network validator pass, should allow creation", func() {
+		vmiBytes, _ := json.Marshal(api.NewMinimalVMI("testvmi"))
+		ar := &admissionv1.AdmissionReview{
+			Request: &admissionv1.AdmissionRequest{
+				Resource: webhooks.VirtualMachineInstanceGroupVersionResource,
+				Object: runtime.RawExtension{
+					Raw: vmiBytes,
+				},
+			},
+		}
```

Comment:
There is an effort to replace in place instantiation of AdmissionReview objects with `newAdmissionReviewForVMICreation`, and api.NewMinimalVMI with libvmi.
Please see https://github.com/kubevirt/kubevirt/pull/13136

Same bellow.

Reply:
Nice, done.

Reply:
Done